app/move/views/confirmation.njk

```diff
@@ -0,0 +1,65 @@
+{% extends "layouts/base.njk" %}
+
+{% block customGtagConfig %}
+  gtag('set', {'page_title': 'Move completed'});
+{% endblock %}
+
+{% block pageTitle %}
+  {{ t("moves::confirmation.page_title", { name: fullname | upper }) }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ super() }}
+
+  {{ govukBackLink({
+    text: t("actions::back_to_dashboard"),
+    classes: "app-print--hide",
+    href: MOVES_URL
+  }) }}
+
+  <a href="javascript:window.print()" class="app-!-float-right app-print--hide govuk-!-margin-top-3">
+    {{ t("actions::print_move_detail") }}
+  </a>
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      {{ govukPanel({
+        titleText: t('moves::confirmation.panel.heading'),
+        html: t('moves::confirmation.panel.content', {
+          moveReference: moveReference
+        })
+      }) }}
+
+      <p>{{ t("moves::confirmation.detail", {
+          fullname: fullname,
+          location: location,
+          date: date
+         }) | safe }}
+      </p>
+
+      <div class="app-print--hide">
+        <h2 class="govuk-heading-m">
+          {{ t("moves::confirmation.cta_heading") }}
+        </h2>
+        <ul class="govuk-list">
+          <li>
+            <a href="{{ action.createNewMoveUrl }}">
+              {{ t("actions::create_new_move") }}
+            </a>
+          </li>
+          <li>
+            {{ t("actions::or") }}
```

Comment:
I wonder if we need "or" at all. A list signifies _or_ options by default, no?

Or if we want or, maybe it could just be a sentence?

[Create a new more](#) or [return to the dashboard]().

Reply:
One for @lmoney-moj / @Andy-Millar though.

Reply:
spoken with @lmoney-moj and have updated the work as Per above